README.md

```diff
@@ -25,7 +25,9 @@ You can specify either a package (+ version) or a local directory to analyze:
 
 ```
 Usage: pana [<options>] <published package name> [<version>]
-       pana [<options>] --source path <local directory>
+       pana [<options>] --source path/to/local/directory
```

Comment:
I think this is not correct. Source has only two values allowed:
https://github.com/dart-lang/pana/blob/master/bin/pana.dart#L30

The current version of the readme is correct in that `--source path` is the verbatim command followed by a path to local package dir.

Reply:
Hey @isoos, I see what you are trying to say but in original README the `--source path` path was quite confusing that's why I have mentioned to/local/directory(it gives a sort of explanation).

Reply:
I see, What I have done wrong here. It should look like `--source path <to/local/directory>`.

Reply:
Yes, `--source path <to/local/directory>` is correct.